forge/gradle.properties

```diff
@@ -3,17 +3,17 @@ org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 
 # Mod
-mod_version=1.18.2-0.18
+mod_version=1.18.2-1.0
 
 # Minecraft
 mc_version=1.18.2
 mc_version_range=[1.18.2,1.19)
 
 # Loader
-forge_version=1.18.2-40.0.2
-forge_version_range=[40.0.2,)
+forge_version=1.18.2-40.1.60
+forge_version_range=[40.1,)
```

Comment:
Same comment as above for the Forge versions, I don't think anything in this PR needs new Forge methods but correct me if I'm wrong.